src/Fixer/Operator/ConcatSpaceFixer.php

```diff
@@ -77,6 +79,29 @@ public function getDescription()
         return 'Concatenation should be spaced according configuration.';
     }
 
+    public function getFixerDescription()
+    {
+        return new FixerDescription(
+            array(
+                array(
+                    "<?php\n\$foo = 'bar' . 3 . 'baz'.'qux';",
+                    null,
+                ),
+                array(
+                    "<?php\n\$foo = 'bar' . 3 . 'baz'.'qux';",
+                    array('spacing' => 'none'),
+                ),
+                array(
+                    "<?php\n\$foo = 'bar' . 3 . 'baz'.'qux';",
+                    array('spacing' => 'one'),
+                ),
+            ),
+            "Configuration must have one element 'spacing' with value 'none' (default) or 'one'.",
```

Comment:
nice idea to auto generate, but sounds complex and we could go without for first implementation